File: FrontendShared/lib/model/post_list_item.dart
import 'json_field.dart';

class PostListItem {
  final String id;
  final String contentPreview;
  final DateTime created;
  final DateTime updated;

  PostListItem(this.id, this.contentPreview, this.created, this.updated);

  factory PostListItem.fromMap(Map<String, dynamic> json) => PostListItem(
      json[ID],
      json[CONTENT_PREVIEW],
      DateTime.parse(json[CREATED]),
      DateTime.parse(json[UPDATED]));
}

File: WebServicesShared/Models/PostListItem.cs
using System;

namespace Accidis.WebServices.Models
{
	public sealed class PostListItem
	{
		public Guid Id { get; set; }

		public string ContentPreview { get; set; }

		public DateTime Created { get; set; }

		public DateTime Updated { get; set; }
	}
}